src/AssetManager/Controller/ConsoleControllerFactory.php

```diff
@@ -2,18 +2,31 @@
 
 namespace AssetManager\Controller;
 
+use Interop\Container\ContainerInterface;
 use Zend\ServiceManager\FactoryInterface;
 use Zend\ServiceManager\ServiceLocatorInterface;
 
 class ConsoleControllerFactory implements FactoryInterface
 {
-    public function createService(ServiceLocatorInterface $serviceLocator)
+    /**
+     * @inheritDoc
+     */
+    public function __invoke(ContainerInterface $container, $requestedName, array $options = null)
     {
-        $serviceLocator = $serviceLocator->getServiceLocator();
-        $console        = $serviceLocator->get('Console');
-        $assetManager   = $serviceLocator->get('AssetManager\Service\AssetManager');
-        $appConfig      = $serviceLocator->get('config');
+        $console        = $container->get('Console');
```

Comment:
We define both `Console` and `console` in zend-mvc-console; I'd recommend the normalized (lowercase) version for the widest range of compatibility.